client-src/elements/chromedash-feature-filter.js

```diff
@@ -215,35 +215,21 @@ class ChromedashFeatureFilter extends LitElement {
     return [
       ...SHARED_STYLES,
       css`
-      iron-icon {
-        --iron-icon-height: 18px;
-        --iron-icon-width: 18px;
+      sl-icon-button {
+        font-size: 1.6rem;
+        margin: 0 !important;
       }
-      button.searchbutton {
-        padding: 2px 8px;
-        font-size: inherit;
+      sl-input sl-icon-button {
+        font-size: 1.4rem;
```

Comment:
font-size is also set at 1.6rem above?

Reply:
These are two different sizes for two different contexts.  The icons that stand alone are a little bigger, whereas the ones that are part of the search input field are a little smaller to be more similar to the size of the text in the input field.

Reply:
These are two different sizes for two different contexts. The icons that stand alone are a little bigger, whereas the ones that are part of the search input field are a little smaller to be more similar to the size of the text in the input field.

Reply:
These are two different sizes for two different contexts. The icons that stand alone are a little bigger, whereas the ones that are part of the search input field are a little smaller to be more similar to the size of the text in the input field.